sentences/ro/_common.yaml

```diff
@@ -47,18 +47,18 @@ lists:
       - in: "f"
         out: "fahrenheit"
 expansion_rules:
-  area: "{area} [(zona | zonă | regiune)]"
+  area: "{area} [(zona | zonă | regiune | cameră | camera)]"
   brightness: "{brightness} [(la suta | la sută | %)]"
   temperature: "{temperature} [[de] grade] [{temperature_unit}]"
   name: "({name}[ul] | {name})"
   este: "(e | este)"
   modifica: "(schimba | schimbă | modifica | modifică | seteaza | setează)"
   in: "(in | în)"
-  din: "(din | in | în | pentru)"
+  din: "(din | in | în | pentru | de la | de pe )"
```

Comment:
```suggestion
  din: "(din | in | în | pentru | [de] la | de pe )"
```

Reply:
Cred că ar trebui să punem și la prepoziția compusă ”de pe” tot ca la ”[de] la” adica "[de] pe". În sentința "<porneste> [<din>] <area> [toate] <lumina>", area ar putea să fie hol, balcon, terasă și atunci ar suna mai bine  de ex. ”aprinde pe hol toate luminile” sau chiar și pe la să îl punem în paranteze pătrate în expresii precum ”aprinde la baie lumina”. Cam ciudată forma asta inversă dar dacă tot o punem să încercăm să ne gândim la cât mai multe variante posibile.

Reply:
Vroiam să te întreb dacă faci tu modificările pe github sau trebuie să fac din nou request cu sugestiile pe care mi le-ai dat. Nu m-am lămurit încă cum funcționează acest proces.

Reply:
Eu nu pot sa modific commit-ul tau. Pot sa fac eu un commit identic, dar dureaza un pic si n-am timp acum.

Ar fi apreciat un nou commit de la tine in acelasi PR, cu modificarile discutate, pe care sa-l review-uiesc eu si/sau @gabimarchidan, sa-l aprobam si sa-i facem merge.